src/signalo/value_lists/management/commands/populate_vl.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+import json
+import os
+
+from ...models import OfficialSignType
+
+
+class Command(BaseCommand):
+    help = "Populate value lists"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-c", "--clean", type=bool, default=False,
+        )
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        signs = []
+
+        path = os.path.abspath(os.path.dirname(__file__))
+
+        with open(f'{path}/../../data/official-signs.json') as csvfile:
+            data = json.load(csvfile)
+            for row in data:
+                sign_instance = OfficialSignType(**row)
```

Comment:
for the image field, it can be loaded like this: https://stackoverflow.com/a/19037233

from the code I think the settings are already there for Django to correctly managed media files